pkg/env/costmodelenv.go

```diff
@@ -511,7 +518,7 @@ func GetAllocationNodeLabelsEnabled() bool {
 	return GetBool(AllocationNodeLabelsEnabled, true)
 }
 
-var defaultAllocationNodeLabelsIncludeList []string = []string{
```

Comment:
this is an unintentional change from `go fmt`, but it looks like a positive change so I'm keeping it in this pull.